src/pages/vaults/hooks/TokenPrice.ts

```diff
@@ -47,7 +47,7 @@ export function useTokenPrice (): DexTokenPrice {
     // active price for walletTokens based on USDT
     const arbitraryPoolPair = getArbitraryPoolPair(symbol, 'USDT')
 
-    if (symbol === 'USDT') {
+    if (symbol === 'USDT' || symbol === 'USDC' || symbol === 'DUSD') {
```

Comment:
```suggestion
    if (symbol === 'USDT') {
```

Reply:
Informed you wrongly.

Reply:
Yes `USDC` can do below one and we can omit it here but what about `DUSD`? why should we remove it from here?

Reply:
Because right now DUSD is not $1 yet. If we fixed it at $1 here the values will be slightly off.

Reply:
The calculations here are based off USDT because its the "most stable" and for simplicity sake.

Reply:
ignore the comment. i can see `dusd` can go below `1`